src/ResourceManager/Profile/Commands.Profile/help/Send-Feedback.md

```diff
@@ -38,7 +38,7 @@ Response.
 Please enter your email if you are interested in providing follow up information:
 
 your@email.com
-```
```

Comment:
@sankethka since you created a public branch in the repository, I was able to push a commit that reverted the changes to the backticks so they appear in this pull request 😄